.github/workflows/generate-clients.yaml

```diff
@@ -58,7 +58,8 @@ jobs:
             --git-repo-id qovery-client-${{ matrix.language }} \
             --package-name qovery \
             --additional-properties packageVersion=$(grep 'version' openapi.yaml | head -1 | tr ':' '\n' | tail -1 | tr -d ' ') \
-            --additional-properties npmVersion=$(grep 'version' openapi.yaml | head -1 | tr ':' '\n' | tail -1 | tr -d ' ')
+            --additional-properties npmVersion=$(grep 'version' openapi.yaml | head -1 | tr ':' '\n' | tail -1 | tr -d ' ') \
+            --additional-properties axiosVersion=1.8.2
```

Comment:
There is a config file to set additional properties https://github.com/Qovery/qovery-openapi-spec/blob/main/generator/configs/typescript-axios.yaml

Reply:
https://github.com/Qovery/qovery-openapi-spec/blob/main/generator/configs/typescript-axios-ws.yaml